src/components/routes/Host/ListingForm/ListingForm.container.ts

```diff
@@ -37,21 +37,22 @@ const ListingFormContainer = styled.section`
           > * {
             margin-bottom: 16px;
           }
-          .row-address {
-            display: flex;
-            flex-direction: row;
-            justify-content: space-between;
-            .bee-input-wrapper {
-              &:nth-of-type(1) {
-                width: 45%;
-              }
-              &:nth-of-type(2) {
-              width: 15%;
-              }
-              &:nth-of-type(3) {
-                width: 20%;
-              }
-            }
+          .city {
+            display: inline-block;
+            margin-bottom: 0;
+            width: 45%;
+          }
+          .state {
+            display: inline-block;
+            margin-bottom: 0;
+            margin-left: 53px;
```

Comment:
i think these margin lefts should be `10%` each to fill out the remaining space

Reply:
mah man 👍